docs/3.0.0-beta.x/guides/registering-a-field-in-admin.md

```diff
@@ -40,21 +40,99 @@ Once this step is over all we need to do is to create our new WYSIWYG which will
 
 ### Creating the WYSIWYG
 
-In this part we will create two components:
+In this part we will create three components:
 
+- MediaLib which will be used to insert media in the editor
 - Wysiwyg which will wrap the CKEditor with a label and the errors
 - CKEditor which will be the implementation of the new WYSIWYG
 
+### Creating the MediaLib
+
+**Path —** `./plugins/wysiwyg/admin/src/components/MediaLib/index.js`
+
+```js
+import React, { useEffect, useState } from 'react';
+import { useStrapi, prefixFileUrlWithBackendUrl } from 'strapi-helper-plugin';
+import PropTypes from 'prop-types';
+
+const MediaLib = ({ isOpen, onChange, onToggle }) => {
+  const {
+    strapi: {
+      componentApi: { getComponent },
+    },
+  } = useStrapi();
+  const [data, setData] = useState(null);
+  const [isDisplayed, setIsDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (isOpen) {
+      setIsDisplayed(true);
+    }
+  }, [isOpen]);
+
+  const Component = getComponent('media-library').Component;
```

Comment:
You can precise the component name. Something like `MediaLibraryComponent`.

Reply:
Media library is the name of the component